src/redux/store/index.ts

```diff
@@ -0,0 +1,12 @@
+import { createStore, applyMiddleware } from "redux";
+import thunkMiddleware from "redux-thunk";
+import rootReducer from "../reducers";
+import { composeWithDevTools } from "redux-devtools-extension";
+
+const store = createStore(
+  rootReducer,
+  composeWithDevTools(applyMiddleware(thunkMiddleware)) //composeWithDevTools is used here for development purposes. It should be removed for production
```

Comment:
i think we should apply check here on `environment` variable that we will get form .env . if production then we will remove `composeWithDevTools` we always forget that we need to remove this for production. what do you think.